### DISCUSSION TITLE
elasticsearch on Hive master

### DISCUSSION THREAD

**Discussion by t0b0rX0r** [telekom-security/tpotce#1103] [General]
Hi, I recently deployed the latest version of T-Pot and I'm trying to get the Hive and Hive sensor setup built out.  After repeated attempts I noticed that the elasticsearch service doesnt seem to be running/installed on the Hive master.  Is this by design?  How would I bring up a Web GUI on the Hive master to view the sensor data?  Am I missing something?

...Any help would be very appreciated!

**Output of dps.sh below from Hive Master**
fatt                  Up 2 hours             
heralding             Up 2 hours             0.0.0.0:110->110/tcp, 0.0.0.0:143->143/tcp, 0.0.0.0:465->465/tcp, 0.0.0.0:993->993/tcp, 0.0.0.0:995->995/tcp, 0.0.0.0:1080->1080
/tcp, 0.0.0.0:5432->5432/tcp, 0.0.0.0:5900->5900/tcp
honeytrap             Up 2 hours             
ipphoney              Up 2 hours             0.0.0.0:631->631/tcp
kibana                Up 2 hours (healthy)   127.0.0.1:64296->5601/tcp
logstash              Up 2 hours (healthy)   
mailoney              Up 2 hours             0.0.0.0:25->25/tcp
map_data              Up 2 hours             
map_redis             Up 2 hours             
map_web               Up 2 hours             127.0.0.1:64299->64299/tcp
medpot                Up 2 hours             0.0.0.0:2575->2575/tcp
nginx                 Up 2 hours             
p0f                   Up 2 hours             
redishoneypot         Up 2 hours             0.0.0.0:6379->6379/tcp
sentrypeer            Up 2 hours             0.0.0.0:5060->5060/udp
snare                 Up 2 hours             0.0.0.0:80->80/tcp
spiderfoot            Up 2 hours (healthy)   127.0.0.1:64303->8080/tcp
suricata              Up 2 hours             
tanner                Up 2 hours             
tanner_api            Up 2 hours             
tanner_phpox          Up 2 hours             
tanner_redis          Up 2 hours

**Comment by t3chn0m4g3** [comment]
If you haven't made changes to the configs it seems some images have not been downloaded. Maybe you ran into [Known Issues](https://github.com/telekom-security/tpotce#docker-images-fail-to-download).

**Reply by t0b0rX0r** [reply]
Thank You, I believe this was the issue.  Subsequent installations resolved the issue
